2018/ffs-wp.md

```diff
@@ -0,0 +1,277 @@
+einzusteigen, zwei Anmerkungen:
+
+- In vielen anderen Bereichen sind Menschen aber bereit, erhebliche Mittel "entgegen der ökonomischen Vernunft" auszugeben, Stichwort: Statuskonsum.
+- Wie oben schon erwähnt, kann die finanzielle Unterstützung eines Projektes von dem man individuell profitiert auch mit informiertem Eigeninteresse begründet werden.
+
+
+### Kombinationsmodell
+Es gibt eine Organisation (Arbeitstitel: "Funding Freedom Society" (FFS)).
+Bei dieser sind alle an Finananzierung interessierten FLOSS-Projekte registriert und haben ein "Konto".
+
+Als Endbenutzer\*in *kauft* man ein Free-Software-Support-Zertifikat (Bronze, Silber, Gold, Platin (z.B. 10, 30, 100, 200 Cent pro Tag)).
+
+**Warum sollte man für sowas Geld ausgeben?**
+
+- Man unterstützt Freie Software ("Voluntary Donation")
+- Man bekommt Merchandise-Material ("Selling branded"), z.B. Aufkleber, schickes T-Shirt
+    - Zusätzlich sind "virtuelle Accessoires" denkbar, also z.b. Badges für github oder andere Soziale Netzwerke
+    Hintergrund: Das geht in Richtung Statuskonsum bzw. [Distinktionsbedürfnis](https://de.wikipedia.org/wiki/Distinktion_(Soziologie)).
+    Man kann dezent zeigen, dass man es sich leisten kann und will, Freie Software zu unterstützten.
+    Dass viele (oder alle) Menschen grundsätzlich ein solches Bedürfnis haben, sieht man an Verkauf von Fan- und Luxus-Artikeln, an Subkultur-spezifischer Kleidung, an personalisierten Notebooks etc.
+- Man bekommt privilegierten Kommunikationskanal ins Projektteam (z.B. höhres Gewicht bei Feature-Wunsch-Abstimmungen)
+- Man bekommt ggf. kleine extra-Features (Für Projekte, die das wollen)
+
+**Wie erfolgt die Verteilung des Geldes?**
+Abegerechnet wird in bestimmten Intervallen (z.b. Quartale).
+Der Zertifikatspreis wird in vier Kategorien aufgeteilt:
+
+- K1: (z.B. 60%) werden nach Kundenwunsch auf die Projektkonten verteilt
+    - Als Entscheidungshilfe wird eine Software angeboten (opt-in), die lokal das Nutzungsverhalten analysiert und Vorschläge macht (Prämisse: Häufig laufende Programme sind subjektiv wichtiger)
+    - Ähnlich zum Pupularity contest des Debian-Projektes
+- K2: (z.B. 30%) werden nach Einschätzung der Organisation auf die Projektkonten verteilt.
+    - Prämisse: FFS hat besseren Überblick über die Projekte und kann ggf. auch strategisch Projekte mit Potential fördern
+- K3: (z.B. 10%) sind Betriebskosten (Personal und Infrastruktur der FFS)
+- K4: Materialkosten (für T-Shirts, etc) wird zusätzlich berechnet
+
+
+Die Zuteilung der Gelder wird "crypto-transparent" veröffentlicht:
+
+- Beim Bezahlen bekommt man einen Token.
+- Alle verkauften Support-Zertifikate werden veröffentlicht. Enthaltene Daten:
+    - Token
+    - Gesamtsumme
+    - K1-Zuteilungsschema
+- -> Jede\*r Käufer\*in kann sich überzeugen, ob das eigene Zertifikat korrekt in der Gesamtbilanz enthalten ist (partielle Geldflüsse).
+- Alle können sich überzeugen, dass die Gesamtbilanz aufgeht.
```

Comment:
s.a. SeaNaps 2017 in Leipzig https://media.c3d2.de/u/vv01f/m/seanaps/

Reply:
Falls Du dazu was im Text haben willst oder auf einen bestimmten Punkt hinaus willst brauche ich mehr Detail-Infos. Nur meine generelle Herangehensweise: Ich habe das Thema Krypto-Währungen erstmal außen vor gelassen, weil es zusätzliche Fragen aufwirft, aber für die Idee des Konzeptes nicht entscheidend ist.

Reply:
Ja, es ging mir darum mal ein konkretes Bsp. zu nennen, das ggf. herangezogen werden kann und nicht meilenweit gegen den Wind nach Betrug (ICO) riecht.

Reply:
Ok.